app/scripts/background.js

```diff
@@ -261,18 +261,35 @@ function maybeDetectPhishing(theController) {
       }
 
       theController.phishingController.maybeUpdateState();
-      const phishingTestResponse =
-        theController.phishingController.test(hostname);
-      if (!phishingTestResponse?.result) {
+      const phishingTestResponse = theController.phishingController.test(
+        details.url,
+      );
+
+      const blockedRequestResponse =
+        theController.phishingController.isBlockedRequest(details.url);
+
+      // if the request is not blocked, and the phishing test is not blocked, return and don't show the phishing screen
+      if (!phishingTestResponse?.result && !blockedRequestResponse.result) {
         return {};
       }
 
+      // Determine the block reason based on the type
+      let blockReason;
+      if (phishingTestResponse?.result && blockedRequestResponse.result) {
+        blockReason = `${phishingTestResponse.type} and ${blockedRequestResponse.type}`;
```

Comment:
Should this be translatable?

Reply:
Hey sorry what do you mean by translatable?

Reply:
Most strings in the extension use a translate function to get the actual string from [one of the messages files](https://github.com/MetaMask/metamask-extension/tree/e8273d87728b342d5cedaf107d354ea2c8672649/app/_locales), depending on the user's selected locale. Example:

https://github.com/MetaMask/metamask-extension/blob/e8273d87728b342d5cedaf107d354ea2c8672649/shared/lib/error-utils.js#L73-L80

I'm not sure if it's possible in this case actually, since this code is not in the usual React flow. 🤔

Reply:
I dont think this is possible as this type is how we blocked the domain. either 'fuzzy', 'blocklist', 'c2DomainBlocklist' so should never be needed to be transalatable

Reply:
Ah I see now. This is only used for metrics tracking?

Reply:
yes! only used for metrics tracking to see why we blocked a specific domain